### PR TITLE
feat: use setValue() on select element

### PR DESCRIPTION
**Pull request by VictorCazanave**
- Resolve https://github.com/vuejs/vue-test-utils/issues/775
- Add error message when calling setValue() on option element
- Fix typo on contributing guide.

### CODE REVIEW

File: packages/test-utils/src/wrapper.js [hunk 1]
```diff
@@ -697,8 +697,12 @@ export default class Wrapper implements BaseWrapper {
     const type = this.attributes().type
 
     if (tagName === 'SELECT') {
+      // $FlowIgnore
+      this.element.value = value
+      this.trigger('change')
+    } else if (tagName === 'OPTION') {
       throwError(
-        `wrapper.setValue() cannot be called on a <select> ` +
+        `wrapper.setValue() cannot be called on a <option> ` +
```
Comment (eddyerburgh) [inline packages/test-utils/src/wrapper.js]: Can you change `a` to `an`—`cannot be called on an <option>`

**Review by eddyerburgh — CHANGES_REQUESTED** [review]
Looks great, just one tiny grammatical change and I'll merge 👍

**Comment by VictorCazanave** [comment]
Thanks! It's fixed 👌

**Review by eddyerburgh — APPROVED** [review]
Great, thanks 👍

**Pull request merged**